src/ophys_etl/modules/segmentation/qc/merge.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
+import networkx as nx
+from matplotlib import cm
+from matplotlib.figure import Figure
+from typing import List
+
+from ophys_etl.modules.segmentation.qc_utils import roi_utils
+from ophys_etl.types import ExtractROI
+
+
+def roi_ancestor_gallery(figure: Figure,
+                         metric_image: np.ndarray,
```

Comment:
It looks like you you pass in a metric image and ROI, all in global field of view coordinates, and the gallery is made up of full field of view thumbnails. Can you add functionality so that the gallery plots only show as many pixels as are necessary to encompass all of the ancestors of the desired merger tree (plus a buffer)?

Maybe just keep track of the extremal (x0+width) and (y0+height) for each ROI being plot and then call
```
ax.set_xlim(xmin-5, xmax+5)
ax.set_ylim(ymin-5, ymax+5)
```